docs/src/rdm.md

```diff
@@ -1,12 +1,12 @@
 # Racing Diffusion Model
 
-The Diffusion Race Model (DRM; Tillman, Van Zandt, & Logan, 2020) is a sequential sampling model in which evidence for options races independently. The DRM is similar to the Linear Ballistic Accumulator, except it assumes noise occurs during the within-trial evidence accumulation process, but the drift rate is constant across trials. 
+The DRacing Diffusion Model (RDM; Tillman, [Van Zandt, & Logan, 2020](https://link.springer.com/article/10.3758/s13423-020-01719-6)) is a sequential sampling model in which evidence for options races independently. The DRM is similar to the Linear Ballistic Accumulator, except it assumes noise occurs during the within-trial evidence accumulation process, but the drift rate is constant across trials.
```

Comment:
Thanks for catching my error. 

It looks like your correction has an extraneous `D`